database/sql/routines/pr_cadastrar_turma_especializacao.sql

```diff
@@ -0,0 +1,19 @@
+-- Autor: Jonathan Braian Dias Vaz (RA:790780)
+-- Este PROCEDIMENTO cadastra as turmas de especialização
+
+CREATE OR REPLACE PROCEDURE pr_cadastra_turmas_especializacao (
+  p_id_disc_especilizacao INTEGER,
+  p_codigo_turma INTEGER,
+  p_data_inicio  DATE,
+  p_data_fim DATE, 
```

Comment:
Essa linha gerou o seguinte erro:

> ERROR:  syntax error at or near ")"

```suggestion
  p_data_fim DATE
```